EIPS/eip-1949.md

```diff
@@ -0,0 +1,138 @@
+---
+eip: 1949
+title: Delayed Distributed Minting
+author: Johann Barbie (@johannbarbie), Pinkiebell (@pinkiebell)
+discussions-to: https://ethresear.ch/t/a-distributed-breeding-function/5264
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-04-18
+requires: 721, ERC1948
```

Comment:
Just realised that #1948 is not merged yet, it would be preferable to merge that first.